html-forms/index.html

```diff
@@ -6,35 +6,25 @@
   <body>
     <h1> Coding Form!</h1>
     <form>
-      <label for="email">
+      <label>
```

Comment:
This `label` shouldn't be wrapped around everything in the form, each of these `input` elements should have it's own label associated with it.